src/engine/renderer/tr_image_ktx.cpp

```diff
@@ -46,7 +46,7 @@ struct KTX_header_t {
 
 const byte KTX_identifier[12]{ 0xAB, 0x4B, 0x54, 0x58, 0x20, 0x31, 0x31, 0xBB, 0x0D, 0x0A, 0x1A, 0x0A };
 const uint32_t KTX_endianness{ 0x04030201U };
-const uint32_t KTX_endianness_reverse{ 0x01020304U };
+const uint32_t KTX_endianness_reverse{ 0x01020304u };
```

Comment:
How is a lowercase 'u' supposed to be cleaner?

Reply:
Unified to 'U', thanks.